src/components/mobile/mobile-toolbar.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="x-list x-list--horizontal x-list--align-center x-list__item--expand">
-    <span v-if="$x.totalResults" class="x-list__item--expand x-font-size--05">
+    <span v-if="$x.totalResults" class="x-title1 x-list__item--expand">
```

Comment:
issue: I don't believe this is the intended style:
![image](https://user-images.githubusercontent.com/68222542/210049328-903f461c-fbd6-4b2a-a935-b2219a2cce49.png)

Reply:
Leftover test I believe